projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## HEAD (unreleased)
 
+- Fix (`ngx-json-object-node-flat`): Remove pre-check for schema property when updating a property name. schemaBuilderMode determines which schema to update, schemaRef or schema
```

Comment:
Small thing but would suggest wrapping the code-specific terminology in code formatting so `schemaBuilderMode`, `schemaRef` and `schema` (only at the end), assuming I have all of those correct.

Reply:
Great catch, thanks. Updated.